### ISSUE TITLE
lt2: add more detail und background zu softmax distribution

### ISSUE THREAD

**Issue by berndbischl** [slds-lmu/lecture_i2ml#997]

**Comment by lisa-wm** [comment]
I'd also touch upon softmax temperature here

**Comment by lisa-wm** [comment]
migration from related issue on same topic:

BB: https://en.wikipedia.org/wiki/Softmax_function
wir brauchen die definition, eigenschaften. und sollten dann zeigen wie das mit log regr, lda zusammenhängt.
und was der verlust ist --> cross entropy
siehe alplaydin 10.7.2. multiple classes, seite 254ff
